monai/networks/blocks/upsample.py

```diff
@@ -106,6 +106,27 @@ def __init__(
                     bias=bias,
                 ),
             )
+        elif up_mode == UpsampleMode.DECONVGROUP:
```

Comment:
please add test cases to this file: https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_upsample_block.py#L21-L58

and with these changes, the docstring is no longer true https://github.com/Project-MONAI/MONAI/blob/2f16f0128124e0fa72d82484e4fc0b1c161f10f9/monai/networks/blocks/upsample.py#L58

Reply:
ok, I did that. thanks!